src/Umbraco.Web.UI.Client/src/packages/core/components/input-slider/input-slider.stories.ts

```diff
@@ -1,16 +1,25 @@
+import type { UmbInputSliderElement } from './input-slider.element.js';
 import type { Meta, StoryObj } from '@storybook/web-components-vite';
+import { html } from '@umbraco-cms/backoffice/external/lit';
 import './input-slider.element.js';
-import type { UmbInputSliderElement } from './input-slider.element.js';
 
 const meta: Meta<UmbInputSliderElement> = {
-	title: 'Components/Inputs/Slider',
+	title: 'Generic Components/Inputs/Slider',
 	component: 'umb-input-slider',
+	render: (args) =>
+		html`<umb-input-slider
+			.min="${args.min}"
+			.max="${args.max}"
+			.step="${args.step}"
+			.valueLow="${args.valueLow}"
+			.valueHigh="${args.valueHigh}"
+			?enableRange="${args.enableRange}"></umb-input-slider>`,
```

Comment:
The [spreadDirective](https://github.com/umbraco/Umbraco.UI/blob/8e7b851c579c0d0ce7ad013ba7d4eab75ff0afb8/storyhelpers/spread-directive.ts) from the UI library should help alleviate this, so we don't have to type all arguments.

```js
html`<umb-input-slider ${spread(args)}></umb-input-slider>`
```

What do you think, should we use that?

Reply:
Good idea. Lets bring it in for a round 2, then we can get this monster merged first.